bindings/spt/block.c

```diff
@@ -47,7 +47,10 @@ solo5_result_t solo5_block_read(solo5_off_t offset, uint8_t *buf, size_t size)
 {
     assert(block_fd >= 0);
 
-    /* XXX: These need hard checks on the tender/seccomp side? */
+    /*
+     * Note that writes beyond capacity are additionally enforced by the
+     * tender's seccomp policy.
+     */
     if(size != block_size)
```

Comment:
Not quite. The Solo5 API should return `SOLO5_R_EINVAL` if you pass invalid arguments. If this check wasn't here then it would just kill the guest with `SIGSYS` due to the seccomp filter violation. While that does the job, it makes it a pain to debug what might just be a programmer error.